values.yaml

```diff
@@ -0,0 +1,26 @@
+# https://github.com/kubernetes/ingress-nginx/blob/helm-chart-4.0.13/charts/ingress-nginx/values.yaml
+controller:
+  resources:
+    requests:
+      cpu: 250m
+      memory: 512Mi
```

Comment:
How do we calculate CPU and memory mean values?

Reply:
they look like a bit high, but my thesis is not supported by data :)

Reply:
We can use something _lighter_ here and eventually increase these values project by project.